spreadsheet/formula/prefixhorizontalrange.go

```diff
@@ -58,3 +58,13 @@ func (r PrefixHorizontalRange) Reference(ctx Context, ev Evaluator) Reference {
 	pfx := r.pfx.Reference(ctx, ev)
 	return Reference{Type: ReferenceTypeHorizontalRange, Value: r.horizontalRangeReference(pfx.Value)}
 }
+
+// String returns a string representation of a horizontal range with prefix.
+func (r PrefixHorizontalRange) String() string {
+	return fmt.Sprintf("%s!%d:%d", r.pfx.String(), r.rowFrom, r.rowTo)
```

Comment:
does this kind of string have any name in Excel?  Just any string that can be input to an excel formula as argument?  I.e. an "expression string" ?  I guess if that is the case the String() method makes sense and fits with the interface name

Reply:
String() method is the opposite to ParseString() method. ParseString() gets Expression type and arguments from a string, and String() makes an expression string back from expression type and arguments.